shared/login/signup/request-invite.js

```diff
@@ -38,12 +50,16 @@ class RequestInvite extends Component {
   }
 }
 
-// $FlowIssue type this connector
 export default connect(
   state => ({
     emailErrorText: state.signup.emailError,
     nameErrorText: state.signup.nameError,
     waiting: state.signup.waiting,
   }),
-  dispatch => bindActionCreators(signupActions, dispatch)
+  (dispatch: Dispatch<*>) => ({
+    // $FlowIssue with thunks
```

Comment:
the normal `Dispatch` type from redux is looking for a function that returns an object, but thunks return another fn